tasks/main.yml

```diff
@@ -5,6 +5,13 @@
   include: floating_ip.yml
   when: (COMMON_FLOATING_INTERFACE_NAME and COMMON_FLOATING_IP)
 
+- name: Add fallback DNS servers to resolve.conf.
```

Comment:
@itsjeyd Yeah, you are right, this would be less confusing (and I mistyped `resolv.conf` anyway).  My original goal was to get the servers into `/etc/resolv.conf`, but that's only reasonably possible via the DHCP client config.